SocialFilter/sections/specific.txt

```diff
@@ -2701,7 +2701,7 @@ odnaminyta.com,sensatsiya.com##.ss-shr
 openrice.com##span[data-href="#share"]
 t.ks.ua##div[class^="sharethis"]
 tabloid.pravda.com.ua##.post__social__side
-all-stars.jp,pravda.com.ua##.post_social
+crescence-studio.itch.io,all-stars.jp,pravda.com.ua##.post_social
```

Comment:
Could you leave just `itch.io`, because it's relevant for all similar posts?

Examples:
- https://gbpatch.itch.io/our-life/devlog/640270/cove-step-1-plushie-now-available-for-limited-time
- https://robobarbie.itch.io/blooming-panic/devlog/401064/ok-i-made-the-walkthroughs
- https://brianna-lei.itch.io/butterfly-soup/devlog/429963/butterfly-soup-2-is-coming-out-on-oct-29
- https://adamgryu.itch.io/a-short-hike/devlog/221497/version-1814-brazilian-portuguese-support

Reply:
@rakleed Didn't notice that, fixed now.